examples/metrics/metrics/observer.js

```diff
@@ -19,7 +19,6 @@ const meter = new MeterProvider({
 
 const otelCpuUsage = meter.createObserver('metric_observer', {
```

Comment:
out of curiosity, how am I able to create a labels at this moment after this change. What would be a correct way of doing it now, if for any reason I want to have labels ?

Reply:
You can't. Labels are now applied only by binding them to an instrument, or calling the instrument method with labels as the second argument like `add(10, { core: 1 })`